org-configurations/default-configuration.yml

```diff
@@ -192,9 +192,9 @@ client:
     status_page:
       content:
         en: |
-           WiFi Login Successful!
-           You can now use the internet.
-           You may leave this page open in case you want to log out.
+          WiFi Login Successful!
+          You can now use the internet.
+          You may leave this page open in case you want to log out.
```

Comment:
Revert these changes please.

Reply:
I still see this request not being met

Reply:
Actually, I have prettier installed, and it tries to maintain uniformity in `.json` and `.yml` files. Shouldn't we keep this change since we're using double quotes everywhere in this file other than these few spots?

Reply:
@someshkar double quotes is fine, but you have deleted two lines in the content section of status page. Please revert that.